tests/classes/CommonTest.php

```diff
@@ -20,14 +20,17 @@ public function testDispositionArray(): void
         $this->assertEquals('none', Common::$disposition[2]);
     }
 
+    /**
+     * @requires PHP >= 8.1
+     */
     public function testCsvConverting(): void
     {
         $this->assertEquals(
             'qwe' . "\n\r" . '' . "\n\r" . '1,qwe,"q w e","""q""w"""' . "\n\r",
             Common::arrayToCSV([
                 'qwe',
                 '',
-                [ 1, 'qwe', 'q w e', '"q"w"' ],
+                [1, 'qwe', 'q w e', '"q"w"'],
```

Comment:
Is this fix really necessary?

Reply:
that's probably phpstorm trying to be helpful with indentation, and no, not necessary.